src/Annotation/Parser/OptionParserTools.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Framework3\Annotation\Parser;
+
+use Framework3\Config\ConfigBag;
+use Framework3\Exception\AppException;
+
+class OptionParserTools
+{
+    /**
+     * @var array
+     */
+    private array $config;
+
+    /**
+     * @param ConfigBag $config
+     *
+     * @throws AppException
+     */
+    public function __construct(ConfigBag $config)
+    {
+        $this->config = $config->get('CONFIG', true);
+    }
+
+    /**
+     * Checks if the annotation option key name matches width the config regex.
+     *
+     * @param string $key
+     *
+     * @return bool
+     *
+     * @throws AppException
+     */
+    public function isOptionKeyValid(string $key): bool
+    {
+        return preg_match($this->getOptionKeyValidRegex(), $key) !== 0;
+    }
+
+    /**
+     * Checks if $value is an int value and do not begin width a zero.
+     *
+     * @param string $value
+     *
+     * @return bool
+     */
+    public function isOptionIntVal(string $value): bool
+    {
+        return preg_match('#^[1-9][0-9]*$#', $value) !== 0;
+    }
+
+    /**
+     * Checks if $value matches width bool true|false keywords
+     *
+     * @param string $value
+     *
+     * @return bool
+     */
+    public function isOptionBoolVal(string $value): bool
+    {
+        return preg_match('#^true|false$#', $value) !== 0;
+    }
+
+    /**
+     * Returns typehint int value of $value
+     *
+     * @param string $value
+     *
+     * @return int
+     */
+    public function getOptionIntVal(string $value): int
+    {
+        return intval($value);
```

Comment:
Codacy found an issue: [The use of function intval() is discouraged; use (int) construction instead.](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6283010)